packages/neos-ui-i18n/src/index.js

```diff
@@ -51,8 +51,44 @@ export default class I18n extends Component {
         return [packageKey, sourceName, id];
     }
 
+    substitutePlaceholders(textWithPlaceholders, parameters) {
+        let startOfPlaceholder;
+        while ((startOfPlaceholder = textWithPlaceholders.indexOf('{')) !== -1) {
```

Comment:
This is a copy-paste from existing Ember version of this thing. I didn't not want to tamper with its logic, as it works and tested by its age :)